cli/cmd/completion.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/cmd/helm/require"
+)
+
+const completionDesc = `
+Generate autocompletion script for lokoctl for the specified shell (bash or zsh).
+`
+
+const bashCompDesc = "\nGenerate the autocompletion script for lokoctl for the bash shell. \n\n" +
```

Comment:
Maybe we can use string literal here to make the formatting simpler? https://golang.org/ref/spec#String_literals

Also, it seems we could make that a function, to de-duplicate parts of the text.